exporters/common/src/main/java/io/opentelemetry/exporter/internal/grpc/GrpcExporterUtil.java

```diff
@@ -39,6 +39,9 @@ static void logUnimplemented(Logger logger, String type, @Nullable String fullEr
       case "log":
         envVar = "OTEL_LOGS_EXPORTER";
         break;
+      case "profile":
+        envVar = "OTEL_PROFILES_EXPORTER";
```

Comment:
This is misleading. This value is used below in a log message to provide a user a hint on how to resolve by setting an env var, but there is no such environment variable for profiling.